src/Uno.UI/UI/Xaml/DependencyObjectExtensions.cs

```diff
@@ -400,6 +400,16 @@ internal static IDisposable RegisterParentChangedCallback(this DependencyObject
 			return GetStore(instance).RegisterParentChangedCallback(key, handler);
 		}
 
+		/// <summary>
+		/// Registers to parent changes.
+		/// </summary>
+		/// <param name="instance">The target dependency object</param>
+		/// <param name="key">A key to be passed to the callback parameter.</param>
+		/// <param name="handler">A callback to be called</param>
+		/// <returns>A disposable that cancels the subscription.</returns>
+		internal static void RegisterParentChangedCallbackStrong(this DependencyObject instance, object key, ParentChangedCallback handler)
+			=> GetStore(instance).RegisterParentChangedCallback(key, handler);
```

Comment:
What's the difference with `RegisterParentChangedCallback`? Same code and same comments :)